src/index.ts

```diff
@@ -32,11 +32,13 @@ const _hasOwn = Object.prototype.hasOwnProperty;
  * @template T
  */
 export function map<T>(opt_initial: T | undefined) {
```

Comment:
```suggestion
export const map = <T>(opt_initial: T | undefined) {
```

Apparently, non-arrow functions are now forbidden.